support_files/server.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIICXQIBAAKBgQCroKE3/m+TV+fjlizmKe20d7wie0FC1TfWgrL7YhH7OdMYG8b+
```

Comment:
:/

Reply:
Not sure, what you refer to but this is the private key that also is part of the qiita repository ...

Reply:
I don't think it's the same, perhaps it will be if someone decides to use a not signed, part of a public repo certificate to start a server somewhere ...

Reply:
As always, do you have a solution or improvement? Do you want to open an issue about it?